waiter/src/waiter/core.clj

```diff
@@ -94,6 +94,7 @@
                      "state" [["" :state-all-handler-fn]
                               ["/fallback" :state-fallback-handler-fn]
                               ["/interstitial" :state-interstitial-handler-fn]
+                              ["/instance-startup" :state-instance-startup-handler-fn]
```

Comment:
Maybe call it `scheduling-metrics`?

Reply:
I'm going to do some renaming, and I'll revisit this endpoint name as well. We're using the term `schedule` for the time from requesting an instance to when it first shows up (probably unhealthy). The time from scheduling until it's healthy is the `startup` time. I'm going to relabel the request-until-heathly overall time as the total `launch` time for the instance.